Update README, and bugfixes for running on Windows

DefaultCyberid opened the pull request:
Please check my changes before merging.

Commit:
Fix wrong error printing
- empty line process, skip the loop instead of printing "error".

Commit:
Fix bug running on Windows
- Change file name format, avoid ":" in it.

Commit:
Update README
- Update the windows specific-requirements infomation.

Commit:
Update README
- Update the windows specific-requirements infomation.

Commit:
Merge branch 'main' of https://github.com/DefaultCyberid/pyJuliusAlign

timmahrt commented:
Hi! Thanks for the PR. Sorry, I have had a lot on my plate, and I forgot to get back to this PR.

I'm working on fixing another of my libraries at the moment but will try to come to this PR afterwards.

I still don't have a windows machine, so I can't validate what you wrote in your readme.  If its ok, I'd like to include it next to the previous advice about using cygwin--and users can perhaps try one or both approaches.

I'll try to validate the two bug fixes you included as well.

Thanks for the PR!

timmahrt commented:
I'm sorry that its taken me so long to get around to this PR.  I know this question comes super late, but can you describe the bugs that you encountered? I hesitate to apply these changes, especially when other windows users haven't needed to apply them.

DefaultCyberid commented:
Sorry for responding so late. I'm no longer working on this project, so I can only recall what problems I encountered from the commit messages.
The problem with `juliusAlignment.py` line 181 is that the string returned by `datetime.now()` will contain the `:` character, which is unacceptable to the Windows file system. So I changed the format of the filename.
The problem with `alignFromTextgrid.py` line 294 is that when `line.strip()` is executed, there is a possibility that `line` will become an empty string. So I added an if action to make the process jump directly to the next loop instead of going into an exception that would cause an extra line of useless information printed.
The contents of `README.txt` are from my records of building the project using Powershell. I tried several times to correct my settings/steps, so I thought it would be a good idea to mark it up to make it easier for other developers to use this repository. Some of the notes (e.g. line 128) are from the official website or other documentation.

I'm not sure how other Windows users have gotten through the process without any problems, perhaps they used a Bash environment such as MinTTY to finish the process？ Idk.

I'm not sure if I'm capable to continue refining this PR, due to time and work problems. So if you think my modifications aren't enough or something is not right, you can reject this PR and then revise it your way.

timmahrt commented:
Thank you for the detailed response. I will try to take a look at this over the weekend and will try to get it merged.  Thank you again!
